### PR TITLE
Add org.kde.kcharselect

### PR DESCRIPTION
**Pull request by nicolasfella**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nicolasfella** [comment]
bot, build org.kde.kcharselect

**Comment by nicolasfella** [comment]
bot, build org.kde.kcharselect

**Comment by barthalion** [comment]
You're probably missing `rename-icon` or `rename-desktop-file`.

**Comment by nedrichards** [comment]
@flathub/kde as per, for anything in the org.kde namespace ;-)

File: org.kde.kcharselect.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    "modules": [
+        {
+            "name": "kcharselect",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/22.04.0/src/kcharselect-22.04.0.tar.xz",
+                    "sha256": "e1f86d5d1eae7433ef49339d855caa6f9d847460b29f096c634d786640e261ab"
```
Comment (tsdgeos) [inline org.kde.kcharselect.json:20]: can you add the magic that will make it auto-update when there's a new release? See other Gear stuff like https://github.com/flathub/org.kde.okular/blob/master/org.kde.okular.json#L487

**Comment by nedrichards** [comment]
Also, we should get that OARS patch in upstream if it isn't already, and likely the icon as well.

File: org.kde.kcharselect.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "id": "org.kde.kcharselect",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kcharselect",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
```
Comment (travier) [inline org.kde.kcharselect.json:10]: `--device=dri` most likely needed

**Comment by Mailaender** [comment]
Superseded by https://github.com/flathub/flathub/pull/5188

**Pull request closed without merging**